### PR TITLE
* Add new cluster: washer-controls-cluster

### PR DESCRIPTION
**Pull request by crlonxp**
- washer-controls-cluster.xml

### CODE REVIEW

File: src/app/zap-templates/zcl/data-model/draft/operationstate.xml
Comment (andy31415) [inline src/app/zap-templates/zcl/data-model/draft/operationstate.xml:1]: Why are these in draft instead of chip-clusters, given that we codegen and start using them?

File: examples/all-clusters-app/all-clusters-common/all-clusters-app.matter [hunk 1]
```diff
@@ -2276,6 +2276,117 @@ server cluster ModeSelect = 80 {
...
+  readonly attribute nullable CHAR_STRING phaseList[] = 0;
+  readonly attribute nullable int8u currentPhase = 1;
+  readonly attribute OperationalStateStruct operationalStateList[] = 3;
+  readonly attribute OperationalStateStruct operationalState = 4;
+  readonly attribute ErrorStateStruct operationalError = 5;
+  readonly attribute command_id generatedCommandList[] = 65528;
+  readonly attribute command_id acceptedCommandList[] = 65529;
+  readonly attribute event_id eventList[] = 65530;
+  readonly attribute attrib_id attributeList[] = 65531;
+  readonly attribute bitmap32 featureMap = 65532;
```
Comment (andy31415) [inline examples/all-clusters-app/all-clusters-common/all-clusters-app.matter:2325]: this featuremap seems invalid. Applies to other places too.

File: src/app/clusters/operational-state-server/operational-state-server.cpp [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+void emberAfOperationalStateClusterInitCallback(chip::EndpointId endpoint)
+{
+    emberAfPrintln(EMBER_AF_PRINT_DEBUG, "--------> %s()", __FUNCTION__);
+    return;
+}
+
+void emberAfOperationalStateClusterServerInitCallback(chip::EndpointId endpoint)
+{
+    // TODO
+    // Get from the "real thermostat"
```
Comment (bzbarsky-apple) [inline src/app/clusters/operational-state-server/operational-state-server.cpp:46]: What does this have to do with thermostats?

I strongly suggest breaking out the XML additions into a separate PR, and doing implementation in a PR on top of that.

File: src/app/zap-templates/zcl/data-model/draft/operationstate.xml [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+    <cluster code="0x0060"/>
+    <item name="OperationalStateID"    type="OperationalStateEnum" optional="false"/>
+    <item name="OperationalStateLabel" type="CHAR_STRING" optional="false"/>
+  </struct>
+
+  <struct name="ErrorStateStruct">
+    <cluster code="0x0060"/>
+    <item name="ErrorStateID" type="ErrorStateEnum" optional="false"/>
+    <item name="ErrorStateLabel" type="CHAR_STRING" length="64" optional="false"/>
+    <item name="ErrorStateDetails" type="CHAR_STRING" length="64" optional="false"/>
```
Comment (bzbarsky-apple) [inline src/app/zap-templates/zcl/data-model/draft/operationstate.xml:45]: Neither of these two matches the spec.

Again, please break out the XML + codegen changes into a separate PR where we can quickly iterate on the XML bits to get them matching the spec and in.

File: src/app/clusters/operational-state-server/operational-state-server.cpp (hunk 2 above)
Comment (crlonxp) [reply src/app/clusters/operational-state-server/operational-state-server.cpp:46]: I will remove them

File: src/app/zap-templates/zcl/data-model/draft/operationstate.xml
Comment (crlonxp) [reply src/app/zap-templates/zcl/data-model/draft/operationstate.xml:1]: These clusters are still under being reviewed. Thank you for the comment. I will move them to "chip" folder

File: examples/all-clusters-app/all-clusters-common/all-clusters-app.matter (hunk 1 above)
Comment (andy31415) [reply examples/all-clusters-app/all-clusters-common/all-clusters-app.matter]: oops ... mis-read.

**Review by woody-apple — APPROVED** [review]
